tests/console/validate_lvm_with_raid1.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: RAID1 on LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "y2_module_consoletest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
+
+sub run {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    my $config = get_test_data();
+    $config->{expected_num_devs} = scalar keys %{$config->{disks}};
```

Comment:
I'd treat this hash more like a static thing, but just my opinion here, it is for avoiding other developers overload this hash instead of writing the values in the yaml. Instead of calculate, just to create in the yaml `num_disk=4`, it is redundant info but make it much easier to read, and we don't need to do this operation here, just the subtraction in line 65. But feel free to leave it as it, now it looks good without global variables.